it/mail/chrome/messenger/addressbook/addressBook.properties

```diff
@@ -146,7 +146,7 @@ failedToExportMessageFileAccessDenied=Esportazione rubrica non riuscita, accesso
 
 # For getting authDN for replication using dlg box
 AuthDlgTitle=Replica rubrica LDAP
-AuthDlgDesc=Inserire nome utente e password per accedere al server di directory.
+AuthDlgDesc=Per accedere al server di directory, inserire nome utente e password.
```

Comment:
CORRECTIONS NEEDED. 

The main clause should always be placed before the subordinate clause. The original text in red was just fine.